src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const lines = table.getElementsByTagName('tr');
+const tableBody = table.querySelector('tbody');
+const counter = lines[0].getElementsByTagName('td');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+
+appendColumn.addEventListener('click', () => {
+  if (counter.length < 10) {
```

Comment:
The condition `counter.length < 10` assumes that the number of columns is static. Since columns can be added or removed, it's better to dynamically calculate the number of columns using `lines[0].getElementsByTagName('td').length` inside the event listener.